clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/DcosClientProvider.groovy

```diff
@@ -20,4 +20,8 @@ class DcosClientProvider {
     String key = account.name
     return dcosClients.computeIfAbsent(key, { k -> DCOSClient.getInstance(account.dcosUrl, account.dcosAuthCredentials) })
   }
+
+  DCOS getDcosClient(String account) {
```

Comment:
perhaps accountName? (differentiates it from the other getDcosClient that takes in "account")